dfir_rs/src/util/deploy.rs

```diff
@@ -4,7 +4,7 @@
 use std::cell::RefCell;
 use std::collections::HashMap;
 
-pub use hydro_deploy_integration::*;
+pub use hydro_deploy_integration::{Connection, DeployPorts, InitConfig, ServerPort, *};
```

Comment:
Why is it desirable to both call out individual imports and also keep the `*`?

Reply:
Good catch, can leave it as it was before. was messing with it because rust-analyzer was giving me some spurious errors (probably some cached build issue)